apps/client/src/features/viewers/common/viewerUtils.ts

```diff
@@ -0,0 +1,28 @@
+import { TimerType } from 'ontime-types';
+
+import { TimeManagerType } from '../../../common/models/TimeManager.type';
+import { formatTime } from '../../../common/utils/time';
+
+const formatOptions = {
+  showSeconds: true,
+  format: 'hh:mm:ss a',
+};
+
+type TimerTypeParams = Pick<TimeManagerType, 'timerType' | 'current' | 'elapsed' | 'clock'>;
+
+export function getTimerByType(timerObject?: TimerTypeParams): string | number | null {
+  let timer = null;
+  if (!timerObject) {
+    return timer;
+  }
+
+  if (timerObject.timerType === TimerType.CountDown) {
+    timer = timerObject.current;
+  } else if (timerObject.timerType === TimerType.CountUp) {
+    timer = timerObject?.elapsed ?? 0;
```

Comment:
Small bug here. if the timer is count-up and there is added time. the timer appears to count down.
I am under the impression that the bug is in the backend. Would you mind taking a look=

Reply:
i think the issue is that the 'elapsed' time is calculated against the 'duration' of the timer. Adding time to the timer let's 'elapsed' go negative wich makes it count down in frontend i think.

Reply:
i've fixed the issue in frontend for now by setting elapsed to 0 if it's smaler than 0. Although i think we should takle that issue in backend. I just dont wanna mess with the calculations for now. Maybe you have some thoughts on that.